packages/jest-each/src/bind.js

```diff
@@ -144,3 +152,53 @@ const applyObjectParams = (obj: any, test: Function) => {
 
 const pluralize = (word: string, count: number) =>
   word + (count === 1 ? '' : 's');
+
+const hasOwnProperty = (object: Object, value: string) =>
+  Object.prototype.hasOwnProperty.call(object, value) ||
+  Object.prototype.hasOwnProperty.call(object.constructor.prototype, value);
+
+const getPath = (object: Object, propertyPath: string | Array<string>) => {
```

Comment:
I've inlined this for now to get it working, it shouldn't stay here :)

Reply:
can we just use `lodash.get` or something?

Reply:
I didn't think we were using `lodash` anywhere? This `getPath` algorithm is used in the `toHaveProperty` matcher and offers a nicer returned value than `lodash` as it tracks the last visited object.

Reply:
sticking it in jest-utils should be fine for sharing (although @cpojer hates the util package :D)

Reply:
Should I move it as part of this PR or would it be better in a separate PR?